PatientRegistrationApp/src/App.vue

```diff
@@ -76,25 +76,33 @@
 									</v-toolbar>
 									<v-card flat><SinglePatientPersonalInfo ref="singlepatientpersonalinfo"/></v-card>							
 								</v-stepper-content>
+
+								<!-- Single Patient: Screening Questions -->
+								<v-stepper-content step="5">
+									<v-toolbar flat>
+										<v-toolbar-title class="text-wrap">Please complete the screening questions</v-toolbar-title>
```

Comment:
Two things:
1. Change the text to be "Please complete these screening questions:"
2. Underneath this header, can you include this statement in smaller text? This will help explain why we're asking these. 
-- I would do this the same as in the below photo
"Information collected on this form will be used to document authorization for receipt of vaccines. Your responses will be verified when receiving the vaccine. The information will be shared through the Wisconsin Immunization Registry (WIR) with other health care providers directly involved with the patient to assure completion of the vaccine schedule. Information collected on this form is confidential." 


![image](https://user-images.githubusercontent.com/67079030/106231813-29514200-61b8-11eb-95e6-0c98b48bb916.png)

Reply:
Done